content/en/docs/concepts/_index.md

```diff
@@ -19,9 +19,9 @@ To work with Kubernetes, you use *Kubernetes API objects* to describe your clust
 
 Once you've set your desired state, the *Kubernetes Control Plane* makes the cluster's current state match the desired state via the Pod Lifecycle Event Generator ([PLEG](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/pod-lifecycle-event-generator.md)). To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster:
 
-* The **Kubernetes Master** is a collection of three processes that run on a single node in your cluster, which is designated as the master node. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
-* Each individual non-master node in your cluster runs two processes:
-  * **[kubelet](/docs/admin/kubelet/)**, which communicates with the Kubernetes Master.
+* The **Kubernetes Primary** is a collection of three processes that run on a single node in your cluster, which is designated as the primary node. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
+* Each individual non-primary node in your cluster runs two processes:
```

Comment:
This needs a bit more rewording (see issue #21800): these processes don't actually have to run on a single host.

Reply:
Is _primary_ the preferred term to replace _master_ elsewhere in the docs?

Reply:
> Is _primary_ the preferred term to replace _master_ elsewhere in the docs?

“It depends”. _primary_ pairs well with _replica_, but sometimes it is better to replace _master_ with _control plane_, or other wording (“leader” if it's the result of an election, for example).

Reply:
Well ... IMHO, "Kubernetes Control Plane" is the term we use elsewhere and the preferred replacement.

Reply:
In other places throughout the documentation, the **Master** (I'll refer to this as the **Primary** down below) is discussed as *one piece* of the Kubernetes Control Plane, not a direct 1-to-1 replacement. The Kubelet is also part of the Control Plane but is not part of the Master.

If we were simply to replace the word "master" with the word "Kubernetes Control Plane", then we end up with awkward descriptions such as this at  https://kubernetes.io/docs/concepts/#kubernetes-control-plane

> The various parts of the Kubernetes Control Plane, such as the Kubernetes Master and kubelet processes

becomes the following, which doesn't make any sense:

> The various parts of the Kubernetes Control Plane, such as the Kubernetes Control Plane and kubelet processes

We can of course re-word this to merge the Primary & CP concepts, but we need to be careful not to erase this architectural distinction. The Kubernetes documentation already has a lot of inconsistencies which are frustrating & confusing for new users. See #21755 for a similar example.

In that case, replacing "Master" with "Primary" would be the easy fix, as long the term "Primary" is the agreed-upon term and is used consistently throughout the documentation.

Reply:
BTW, I don't count the kubelet as part of the control plane - see for example the diagram at https://kubernetes.io/docs/concepts/overview/components/#editPageButton

Reply:
I think I'm going to rewrite https://kubernetes.io/docs/concepts/#kubernetes-control-plane anyway.